css/style.css

```diff
@@ -0,0 +1,1544 @@
+    padding-bottom: 20px;
+  }
+
+  .footbottom {
+    display: block;
+    font-size: 90%;
+  }
+
+  .footcont {
+    width: 30%;
+    margin: 0 auto;
+  }
+
+  #tnwfooter {
+    width: 30px;
+    height: 15px;
+  }
+}
+
+/* media query for mobile devices */
+
+@media only screen and (max-width: 599px) {
+  /* navigation small */
+
+  .navtop {
+    display: none;
+  }
+
+  /* navlogom&s style */
+  .navlogom-s {
+    display: block;
+    width: 45px;
+    height: 35px;
+    padding-top: 15px;
+    padding-left: 20px;
+  }
+
+  .navlogom-s img {
+    width: 80%;
+  }
+
+  .navli2 {
+    display: none;
+  }
+
+  /* navigation fixed styling */
+  .header {
+    position: sticky;
+    top: 0;
+    width: 100%;
+    z-index: 100;
+  }
+
+  .navfixed {
+    height: 50px;
+    font-family: "Nova Square", cursive;
+    background-color: white;
+    font-size: 90%;
+  }
+
+  #searchbtn {
+    border-bottom: 1px rgba(84, 84, 84, 0.27) solid;
+    border-top: 1px rgba(84, 84, 84, 0.27) solid;
+    padding-bottom: 10px;
+    padding-left: 15px;
+    padding-top: 15px;
+    width: 35px;
+    font-size: 110%;
+  }
+
+  #hamburger {
+    display: block;
+    color: orangered;
+    margin-left: 10px;
+    font-size: 120%;
+  }
+
+  .hiddenmd {
+    display: none;
+  }
+
+  /* yellow nav styling */
+
+  .yellow-nav {
+    background-color: #ffe712;
+    height: 30px;
+    padding-top: 20px;
+    text-align: center;
+    max-width: 100%;
+    clear: both;
+  }
+
+  .yellow-nav a {
+    font-size: 60%;
+  }
+
+  /* main content */
+
+  .main-news-section {
+    display: grid;
+    grid-gap: 6px;
+    grid-template-areas:
+      "main"
+      "side-top"
+      "side-bottom";
+    margin: 4px 4px 5% 4px;
+  }
+
+  .main-pic {
+    background:
+      linear-gradient(
+        180deg,
+        #f42 0%,
+        rgba(68, 152, 217, 0.11528361344537819) 100%
+      ),
+      url("../images/main-pic-big.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    grid-area: main;
+    height: 35vh;
+    padding: 1% 1% 2% 1%;
+  }
+
+  /* Side pic top */
+  .side-pic-top {
+    height: 35vh;
+  }
+
+  /* side pic bottom */
+  .side-pic-bottom {
+    height: 35vh;
+  }
+
+  /* logo sizing */
+
+  .logo-TNW img {
+    display: none;
+  }
+
+  p {
+    margin: 3px 0;
+  }
+
+  .logo-TNW p {
+    display: none;
+  }
+
+  /* styling the headline texts */
+
+  .pic-content p,
+  .side-pic-top p,
+  .side-pic-bottom p {
+    color: #fff;
+    margin: 0 0 0 10px;
+  }
+
+  .pic-content i {
+    margin: 0 4px;
+  }
+
+  .headline {
+    font-family: "Russo One", sans-serif;
+    font-size: 80%;
+    text-align: justify;
+    padding-top: 115px;
+  }
+
+  #side1 {
+    padding-top: 160px;
+    font-size: 90%;
+  }
+
+  #side2 {
+    padding-top: 140px;
+    font-size: 90%;
+  }
+
+  .titleandlinks {
+    display: flex;
+    flex-wrap: wrap;
+    margin-bottom: 10px;
+  }
+
+  #br {
+    display: none;
+  }
+
+  #br-first {
+    display: none;
+  }
```

Comment:
- [ ] OPTIONAL ➡ Those two selelectors do the same thing, yet they're separated plust you're using ID instead classes. Kindly fix that